pylint/checkers/unicode.py

```diff
@@ -100,7 +100,7 @@ def human_code(self) -> str:
         "\\x1A",
         "E2512",
         (
-            "Ctrl+Z “End of text” on Windows. Some programs (such as type) ignore "
+            'Ctrl+Z "End of text" on Windows. Some programs (such as type) ignore '
             "the rest of the file after it."
```

Comment:
``“`` ?

Reply:
Yeah! I don't know why and how because some times they could print but I think they can't print to the console of Windows?
When I ran the test in the "test explorer" of VS Code on Windows the test even passed, but in the terminal it crashed. This should fix it though 😄